app/views/application/_no_log_requests.html.slim

```diff
@@ -14,6 +14,6 @@ ul.StatsMessages-list
   li The 3scale
     #{link_to('API ActiveDocs documentation', provider_admin_api_docs_path(anchor: '/service_management_api'), target:'_blank')}
     for our Service Management API methods where the logs with response codes can be sent.
-  li Our HowTo tutorial on logs and
-    #{link_to('Response Codes Tracking', I18n.t('api.integrations.proxy.documentation_response_codes_tracking_url'), target:'_blank')}
-    on the 3scale support portal
+  li Follow the documentation about
```

Comment:
Actually.... I can't find where this view is included. Maybe we should delete it?

Reply:
Yes, I'd say we can since nothing is rendering that partial, nice catch!